server/src/test/scala/za/co/absa/atum/server/api/controller/PartitioningControllerUnitTests.scala

```diff
@@ -90,6 +90,20 @@ object PartitioningControllerUnitTests extends ZIOSpecDefault with TestData {
   when(partitioningServiceMock.getPartitioningMainFlow(999L))
     .thenReturn(ZIO.fail(GeneralServiceError("boom!")))
 
+//  when(partitioningServiceMock.patchPartitioningParent(111L, 1111L, "Jack"))
+//    .thenReturn(ZIO.succeed(parentPatchV2DTO01))
+//  when(partitioningServiceMock.patchPartitioningParent(222L, 2222L, "Jill"))
+//    .thenReturn(ZIO.fail(NotFoundServiceError("not found")))
+//  when(partitioningServiceMock.patchPartitioningParent(999L, 2222L, "Bean"))
+//    .thenReturn(ZIO.fail(GeneralServiceError("boom!")))
```

Comment:
consider removing these unrelated bits so that our master stays as clear as possible